voxblox/include/voxblox/test/layer_test_utils.h

```diff
@@ -87,6 +118,246 @@ void LayerTest<TsdfVoxel>::CompareVoxel(const TsdfVoxel& voxel_A,
+        Block<TsdfVoxel>::Ptr block = layer->allocateBlockPtrByIndex(block_idx);
+        TsdfVoxel& voxel = block->getVoxelByLinearIndex(
+            (x * z + y) % layer->voxels_per_side());
+        voxel.distance = x * y * 0.66 + z;
+        voxel.weight = y * z * 0.33 + x;
+        voxel.color.r = static_cast<uint8_t>(x % 255);
+        voxel.color.g = static_cast<uint8_t>(y % 255);
+        voxel.color.b = static_cast<uint8_t>(z % 255);
+        voxel.color.a = static_cast<uint8_t>(x + y % 255);
+
+        block->has_data() = true;
+      }
+    }
+  }
+  double size_in_MB = static_cast<double>(layer->getMemorySize()) * 1e-6;
+  std::cout << std::endl
+            << "Set up a test TSDF layer of size " << size_in_MB << " MB";
+}
+
+template <>
+void SetUpTestLayer(const size_t block_volume_diameter,
+                    Layer<EsdfVoxel>* layer) {
+  CHECK(layer);
+
+  int32_t half_index_range = block_volume_diameter / 2;
+
+  for (int32_t x = -half_index_range; x <= half_index_range; ++x) {
+    for (int32_t y = -half_index_range; y <= half_index_range; ++y) {
+      for (int32_t z = -half_index_range; z <= half_index_range; ++z) {
+        BlockIndex block_idx = {x, y, z};
+        Block<EsdfVoxel>::Ptr block = layer->allocateBlockPtrByIndex(block_idx);
+        EsdfVoxel& voxel = block->getVoxelByLinearIndex(
+            (x * z + y) % layer->voxels_per_side());
+
+        voxel.distance = x * y * 0.66 + z;
+        voxel.parent.x() = x % 255;
+        voxel.parent.y() = y % 255;
+        voxel.parent.z() = z % 255;
+
+        voxel.observed = true;
+        voxel.in_queue = true;
+        voxel.fixed = true;
+
+        block->has_data() = true;
+      }
+    }
+  }
+  double size_in_MB = static_cast<double>(layer->getMemorySize()) * 1e-6;
+  std::cout << std::endl
+            << "Set up a test Esdf layer of size " << size_in_MB << " MB";
+}
+
+template <>
+void SetUpTestLayer(const size_t block_volume_diameter,
+                    Layer<OccupancyVoxel>* layer) {
+  CHECK(layer);
+
+  int32_t half_index_range = block_volume_diameter / 2;
+
+  for (int32_t x = -half_index_range; x <= half_index_range; ++x) {
+    for (int32_t y = -half_index_range; y <= half_index_range; ++y) {
+      for (int32_t z = -half_index_range; z <= half_index_range; ++z) {
+        BlockIndex block_idx = {x, y, z};
+        Block<OccupancyVoxel>::Ptr block =
+            layer->allocateBlockPtrByIndex(block_idx);
+        OccupancyVoxel& voxel = block->getVoxelByLinearIndex(
+            (x * z + y) % layer->voxels_per_side());
+
+        voxel.probability_log = x * y * 0.66 + z;
+        voxel.observed = true;
+
+        block->has_data() = true;
+      }
+    }
+  }
+  double size_in_MB = static_cast<double>(layer->getMemorySize()) * 1e-6;
+  std::cout << std::endl
+            << "Set up a test occupancy layer of size " << size_in_MB << " MB";
+}
+
+template <typename VoxelType>
+bool IsSameVoxel(const VoxelType& voxel_A, const VoxelType& voxel_B) {
```

Comment:
actually rest of voxblox is using maplab style for rest, but sure I agree with this justification. Either way is fine for me. New stuff should preferably use thisStyleOfCamelCase but matching existing style is fine.